war/src/main/java/io/lumeer/engine/rest/SecurityService.java

```diff
@@ -0,0 +1,405 @@
+         throw new BadRequestException();
+
+      organizationFacade.setOrganizationCode(organizationCode);
+   }
+
+   private boolean isRoleCorrect(String resource, String roleName) {
+      return LumeerConst.Security.RESOURCE_ROLES.get(resource).contains(roleName);
+   }
+
+   /**
+    * Add a role for users and groups for an organization.
+    *
+    * @param role
+    *       the role to be added
+    * @param users
+    *       the users to be added the role for
+    * @param groups
+    *       the groups to be added the role for
+    */
+   @PUT
+   @Path("/roles/{role}")
+   public void addOrganizationUsersGroupsRole(
+         final @PathParam("role") String role,
+         final @QueryParam("users") List<String> users,
+         final @QueryParam("groups") List<String> groups) {
+      if (role == null || !isRoleCorrect(LumeerConst.Security.ORGANIZATION_RESOURCE, role))
+         throw new BadRequestException();
+
+      for (String user: users) {
+         securityFacade.addOrganizationUserRole(organizationCode, user, role);
+      }
+
+      for (String group: groups) {
+         securityFacade.addOrganizationGroupRole(organizationCode, group, role);
+      }
+   }
+
+   /**
+    * Get roles of an organization.
+    *
+    * @return
+    *       the role object with all roles for the organization
+    * @throws DataFormatException
+    *       if obtained data cannot be transformed to a role object
+    */
+   @GET
+   @Path("/roles")
+   public List<Role> getOrganizationRoles() throws DataFormatException {
+      return securityFacade.getOrganizationRoles(organizationCode);
+   }
+
+   /**
+    * Delete a role for users and groups for an organization.
+    *
+    * @param role
+    *       the role to be deleted
+    * @param users
+    *       the users to be deleted the role for
+    * @param groups
+    *       the groups to be deleted the role for
+    */
+   @DELETE
+   @Path("/roles/{role}")
+   public void removeOrganizationUsersGroupsRole(
+         final @PathParam("role") String role,
+         final @QueryParam("users") List<String> users,
+         final @QueryParam("groups") List<String> groups) {
+      if (role == null || !isRoleCorrect(LumeerConst.Security.ORGANIZATION_RESOURCE, role))
+         throw new BadRequestException();
+
+
+      for (String user: users) {
+         securityFacade.removeOrganizationUserRole(organizationCode, user, role);
+      }
+
+      for (String group: groups) {
+         securityFacade.removeOrganizationGroupRole(organizationCode, group, role);
+      }
+   }
+
+   /**
+    * Add a role for users and groups for a project.
+    *
+    * @param projectCode
+    *       the project code to be added the role for
+    * @param role
+    *       the role to be added
+    * @param users
+    *       the users to be added the role for
+    * @param groups
+    *       the groups to be added the role for
+    */
+   @PUT
+   @Path("/projects/{project}/roles/{role}")
```

Comment:
@kubedo8 probably not but we should rather set it since we rely on this in many parts of our application. I plan to change this whole mechanism around organization and project context in the future.